pom.xml

```diff
@@ -806,6 +820,12 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.apache.hadoop</groupId>
```

Comment:
I notice there are some excluded transitive dependencies in other hadoop related dependencies. Please make sure this one also does the same thing for those transitive dependencies.

Reply:
@jackjlli Not sure if I understand, what you are saying, completely, but all the dependencies (direct and transitive) are converging with these changes. These are the minimal set of changes that we need to do to upgrade to Avro 1.9.2.

Reply:
E.g.  some of the transitive dependencies of `hadoop-client` got excluded in root pom file, such as `jersey-core`, `protobuf-java`, etc. Make sure those transitive dependencies won't get pulled in again by `hadoop-mapreduce-client-core`.

Reply:
Some of the exclusions that we have in pom files may be redundant :-) But it either case, there are no conflicting dependencies being pulled in because of hadoop-mapreduce-client-core. If a higher-level project includes us and runs into conflicts because of our transitive dependencies (say dependencies being pulled in by the hadoop libraries that we are pulling in) they can always use <exclude>.